app/javascript/styles/application.scss

```diff
@@ -15,5 +15,6 @@ $fa-font-path: "~font-awesome/fonts/";
 @import "includes/tables";
 @import "includes/pagination";
 @import "includes/forms";
+@import "includes/notification";
```

Comment:
import css from npm library instead of copying it to the project

https://github.com/jboesch/Gritter/blob/master/css/jquery.gritter.css

```scss
@import "~gritter/css/jquery.gritter";
```

or you use some custom gritter css? why?